src/Tizen.NET.MaterialComponents/Constants.cs

```diff
@@ -3,6 +3,8 @@
     public static class Styles
     {
         public static readonly string Material = "material";
+        public static readonly string Singleline = "singleline";
```

Comment:
Where is this `Singleline` used? Is this for label?

Reply:
here

https://github.com/TizenAPI/Tizen.NET.MaterialComponents/blob/25ecb9c9ebd42ae14863952e94c0dbf80219a544/src/Tizen.NET.MaterialComponents/Components/MTextField.cs#L150